gift-box-shop-frontend/styles/components/ShopMenu.module.scss

```diff
@@ -0,0 +1,23 @@
+.containerShopMenu{
+    color: grey;
+    width: 10vw;
+    height: 30vh;
+    left: 10vw;
+    top: 4vh;
+    position: relative;
+    font-size: 1.2rem;
+    display: flex;
```

Comment:
There are mixins on the `/styles/utils.scss` files for all the flex properties, use those instead to avoid browser compatibility issues